geomesa-arrow/geomesa-arrow-jts/src/main/java/org/locationtech/geomesa/arrow/jts/GeometryVector.java

```diff
@@ -32,4 +34,7 @@ public interface GeometryVector<T extends Geometry, V extends FieldVector> exten
   int getNullCount();
 
   void transfer(int fromIndex, int toIndex, GeometryVector<T, V> to);
+
+  Map<String, Object> getOptions();
+  void setOptions(Map<String, Object> map);
```

Comment:
as mentioned above, instead of making this so general purpose I think it would be better to just make it a boolean